larndsim/detector_properties/module0.yaml

```diff
@@ -26,35 +27,38 @@ module_to_io_groups:
   1: [1,2,3,4]
```

Comment:
I guess here it should be `1: [1,2]` right?

Reply:
Oh yeah, I guess I missed that

Reply:
Can you change it in this PR?

Reply:
Done